src/Dapr.Actors/Communication/ActorCommunicationException.cs

```diff
@@ -0,0 +1,134 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Actors.Communication
```

Comment:
Is there a good reason to use this namespace? I don't think this is something we expect users to already ahve a `using` for.

Can we put this in the same namespace as all of the other classes that users use when calling into an Actor? It's better to keep related functionality together based on *usage*.